popSimManu.tex

```diff
@@ -647,25 +677,51 @@ \section*{Discussion}
 and demographic histories of different organisms, provides valuable information
 about how methods might perform on non-human systems.
 Finally, comparison of results across methods or simulation runs
-provides an estimate of inference uncertainty, analogous to parametric
-bootstrapping,
-especially since different methods are likely vulnerable to model misspecification
-in different ways.
+provides an estimate of inference uncertainty, analogous to parametric bootstrapping,
+especially when different methods are vulnerable to model misspecification in different ways.
 
+\paragraph{Next steps}
 \Stdpopsim is intended to be a fully open, community-developed project.
 Our implementations of genome representations and genetic maps for the some of
 the most common study systems in computational genetics---humans, \textit{Drosophila},
 and \textit{Arabidopsis} (among others)---are only intended to be a starting point for
-future development.  In addition to other taxa,
-we plan to incorporate other common biological processes
-such as selection, gene conversion, and mutational heterogeneity.
+future development.  
 Researchers are invited to contribute to the resource by adding their
 organisms and models of choice. The \stdpopsim resource is
 accompanied by clearly documented standard operating procedures that are
 intended to minimize barriers to entry for new developers.  In this way, we
 expect the resource to expand and adapt to meet the evolving needs of the
 population genomics community.
 
+One of our goals is to engage research communities studying other taxa,
+so as to expand the resource to many more species.
+Although we have included demographic models and recombination maps,
+there are many biological processes that we do not model.
+Some of these additions that we are enthusiastic to add are:
+selection (including distributions of fitness effects, maps of functional elements, and single sweeps),
+gene conversion, mutation models (rate heterogeneity),
+ancient samples,
```

Comment:
We already have support for sampling in the past. Does this refer to something else?

Reply:
Whoops - that must have been written before we added it. Removed!